### PR TITLE
chore(#23): Hide the secrets from the eyes of the public !

### LINKED ISSUE #23: chore: Remove sensitive informations from the git
The goal is to remove secret informations from this git repository like the `SECRET_KEY` in the settings file... :disappointed:

### PR DESCRIPTION
**Pull request by Narekushia**
Currently, the `SECRET_KEY` used on the production server is commited and everyone is able to see it.

Introducing, environment variable ! :tada: The variable is replaced with a call to an uncommited env file (uses [django-environ](https://github.com/joke2k/django-environ)).

For the devs, there is a `.env.example` available in `sousmot`. Just rename it to `.env` and generate a secret key and everything is good :ok_hand:

For the server, I need to add environment variable to the system before merging.
- [x] <-- This box will be checked when I have done it

Also, I realized that the **Github Action didn't work on merge**... only on push, so I fixed that :+1:  

Fixes #23

### CODE REVIEW

File: sousmot/sousmot/settings.py [hunk 1]
```diff
@@ -10,22 +10,25 @@
...
+# Settings setted in the .env (located alongside the manage.py)
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+SECRET_KEY = env('SOUSMOT_SECRET_KEY')
 
-ALLOWED_HOSTS = []
+DEBUG = env('SOUSMOT_DEBUG_MODE')
 
+ALLOWED_HOSTS = env.list('SOUSMOT_ALLOWED_HOSTS')
```
Comment (MassimoFerro) [inline sousmot/sousmot/settings.py:31]:
```suggestion
ALLOWED_HOSTS = env('SOUSMOT_ALLOWED_HOSTS').split(" ")
```

File: sousmot/.env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SOUSMOT_SECRET_KEY='-------- KEY HERE ---------'
+SOUSMOT_DEBUG_MODE=True
+SOUSMOT_ALLOWED_HOSTS='localhost'
```
Comment (MassimoFerro) [inline sousmot/.env.example]:
```suggestion
SOUSMOT_ALLOWED_HOSTS='localhost 127.0.0.1'
```

**Review by MassimoFerro — COMMENTED** [review]
Maybe we can add 127.0.0.1 as allowed host ? What do you think ?

Otherwise, everything works.

**Comment by Narekushia** [comment]
> Maybe we can add 127.0.0.1 as allowed host ? What do you think ?
> 
> Otherwise, everything works.

Good idea ! :+1: 
It's done in 728d7d8

**Review by MassimoFerro — APPROVED** [review]
Tip-top !

**Pull request merged**